docs/tags.rst

```diff
@@ -114,7 +114,8 @@ The following values are take from the ``BoundField``:
 - id_for_label
 - label
 - name
-- value
+- value (value of the field converted to string)
+- raw_value (e.g. True/False for BooleanField or datetime.datetime for DateTimeField)
```

Comment:
I'd expect `None` to be a possible value as well, and maybe even `on` on submit.